src/sentry/killswitches.py

```diff
@@ -53,19 +53,28 @@ class KillswitchInfo:
         },
     ),
     "store.load-shed-process-event-projects": KillswitchInfo(
-        description="Drop events in process_event.",
+        description="Skip event process_event and forward to save_event",
```

Comment:
Do we actually update the logic for this killswitch somewhere?

Reply:
yes at https://github.com/getsentry/sentry/pull/27827/files#diff-17b5957d1786531cc45705f013bb1e408bca1a086adac7a1338c0cc9bc506eb6R427